great_expectations/render/util.py

```diff
@@ -209,3 +212,219 @@ def handle_strict_min_max(params: dict) -> (str, str):
     )
 
     return at_least_str, at_most_str
+
+
+def build_count_table(
+    partial_unexpected_counts: List[dict], unexpected_count: Optional[int] = None
+) -> Tuple[List[str], List[List[Any]]]:
+    """
+    Used by _diagnostic_unexpected_table_renderer() method in Expectation to render
+    Unexpected Counts table.
+
+    Args:
+        partial_unexpected_counts: list of dictionaries containing unexpected values and counts
+        unexpected_count: total number of unexpected values. Used to build the header
+
+    Returns:
+        List of strings that will be rendered into DataDocs
+
+    """
+    table_rows: List[List[str]] = []
+    total_count: int = 0
+
+    for unexpected_count_dict in partial_unexpected_counts:
+        value: Optional[Any] = unexpected_count_dict.get("value")
+        count: Optional[int] = unexpected_count_dict.get("count")
+        if count:
+            total_count += count
+        if value is not None and value != "":
+            table_rows.append([value, count])
+        elif value == "":
+            table_rows.append(["EMPTY", count])
+        else:
+            table_rows.append(["null", count])
+
+    # Check to see if we have *all* of the unexpected values accounted for. If so,
+    # we show counts. If not then we show Sampled Unexpected Values
+    if total_count == unexpected_count:
+        header_row = ["Unexpected Value", "Count"]
+    else:
+        header_row = ["Sampled Unexpected Values", "Count"]
+    return header_row, table_rows
+
+
+def build_count_and_index_table(
+    partial_unexpected_counts: List[dict],
+    unexpected_index_list: List[dict],
+    unexpected_count: Optional[int] = None,
```

Comment:
It looks like `unexpected_count` is not optional (compared to an int in the function)

Reply:
Thanks for catching this :)

Reply:
made the change in `build_count_table` as well